app/models/financialDetails/TransactionItem.scala

```diff
@@ -47,7 +47,7 @@ trait TransactionItem {
   }
 
   // TODO: duplicate logic, in scope of => https://jira.tools.tax.service.gov.uk/browse/MISUV-8557
-  def getChargeTypeKey(reviewAndReconcileEnabled: Boolean = false): String =
+  def getChargeTypeKey(reviewAndReconcileEnabled: Boolean = false, penaltiesEnabled: Boolean = false): String =
```

Comment:
I am not sure if this method has anything to do with the fact is one or another feature switch is on or off;
might be a good idea to track / isolate this FS check up the calling stack;

Reply:
I think basically we don't want the content for penalties/review&reconcile to show up if the FS are off, and instead they should show "Unknown charge". I think maybe we could include this in the story to remove the duplicated getChargeTypeKey methods

Reply:
I'll add a comment to [MISUV-8557](https://jira.tools.tax.service.gov.uk/browse/MISUV-8557)